bin/mutator-sensu-go-into-v1.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require 'sensu-mutator'
+
+class Helper < Sensu::Mutator
+  def mutate
```

Comment:
lets put a print statement in here to make it more obvious that this is only intended for development purposes.

Reply:
print to stderr?    I think the way mutators work is they output back to stdout.

Reply:
STDERR.puts now in place with a warning message.  I also added a similar message in the mutator test to make sure it didn't screw anything up.